src/Menu/PickerMenuBuilder.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Menu;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\Renderer\RendererInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * Creates the picker menu.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class PickerMenuBuilder
```

Comment:
This is a service that will be either injected or retrieved from the container. It should thus specify a `PickerMenuBuilderInterface` so one could replace it completely. Should be as easy as using phpStorm's Refactor->Extract->Interface functionality ;-)